### PR TITLE
Add documentation on the preview ADB linking experience

### PR DESCRIPTION
**Pull request by akshaya-a**

### CODE REVIEW

File: how-to-use-azureml/azure-databricks/README.md [hunk 1]
```diff
@@ -1,33 +1,73 @@
...
+You can use Azure Databricks as a compute target from [Azure Machine Learning Pipelines](https://docs.microsoft.com/en-us/azure/machine-learning/service/concept-ml-pipelines). Take a look at this notebook for details: [aml-pipelines-use-databricks-as-compute-target.ipynb](https://github.com/Azure/MachineLearningNotebooks/tree/master/how-to-use-azureml/azure-databricks/databricks-as-remote-compute-target/aml-pipelines-use-databricks-as-compute-target.ipynb).
+
+# Linked Azure Databricks and Azure ML Workspaces (Preview)
+Customers can now link Azure Databricks and AzureML Workspaces to better enable MLOps scenarios by [managing their tracking data in a single place when using the MLflow client](https://mlflow.org/docs/latest/tracking.html#mlflow-tracking) - the Azure ML workspace.
+
+## Linking the Workspaces (Admin operation)
+
+1. The Azure Databricks Azure portal blade now includes a new button to link an Azure ML workspace
+![New ADB Portal Link button](./img/adb-link-button.png)
+2. Both a new or existing Azure ML Workspace can be linked in the resulting prompt. Follow any instructions to set up the Azure ML workspace.
```
Comment (akarloff) [inline how-to-use-azureml/azure-databricks/README.md]: Workspace lower case

File: how-to-use-azureml/azure-databricks/README.md [hunk 2]
```diff
@@ -1,33 +1,73 @@
...
+![Link Prompt](./img/link-prompt.png)
+3. After a successful link operation, you should see the Azure Databricks overview reflect the linked status
+![Linked Successfully](./img/adb-successful-link.png)
+
+## Configure MLflow to send data to Azure ML (All roles)
+
+1. Add azureml-mlflow as a library to any notebook or cluster that should send data to Azure ML. You can do this via
```
Comment (akarloff) [inline how-to-use-azureml/azure-databricks/README.md]: via:

File: how-to-use-azureml/azure-databricks/README.md [hunk 3]
```diff
@@ -1,33 +1,73 @@
...
+        ```
+    2. [Cluster Libraries](https://docs.azuredatabricks.net/user-guide/libraries.html#install-a-library-on-a-cluster)
+    ![Cluster Library](./img/cluster-library.png)
+2. [Set the MLflow tracking URI](https://mlflow.org/docs/latest/tracking.html#where-runs-are-recorded) to the following scheme:
+    ```
+    adbazureml://${azuremlRegion}.experiments.azureml.net/history/v1.0/subscriptions/${azuremlSubscriptionId}/resourceGroups/${azuremlResourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/${azuremlWorkspaceName}
+    ```
+    1. You can automatically configure this on your clusters using this helper script
+        1. [AzureML Tracking Cluster Init Script](./linking/README.md)
+3. That's it! If configured correctly, you'll now be able to see your MLflow tracking data in both Azure ML (via the REST API and all clients) and Azure Databricks (in the MLflow UI and using the MLflow client)
```
Comment (akarloff) [inline how-to-use-azureml/azure-databricks/README.md]: IMO remove That's it! I appreciate the flavor text but it's not really necessary for technical documentation.

File: how-to-use-azureml/azure-databricks/README.md [hunk 4]
```diff
@@ -1,33 +1,73 @@
...
+
+
+## Known Preview Limitations
+While we roll this experience out to customers for feedback, there are some known limitations we'd love comments on in addition to any other issues seen in your workflow
```
Comment (akarloff) [inline how-to-use-azureml/azure-databricks/README.md]: workflow.

File: how-to-use-azureml/azure-databricks/README.md [hunk 5]
```diff
@@ -1,33 +1,73 @@
...
+### 1-to-1 Workspace linking
+Currently, an Azure ML Workspace can only be linked to one Azure Databricks Workspace at a time.
+### Data synchronization
+At the moment, data is only generated and sent to Azure ML as well for tracking. Editing tags via the Azure Databricks MLflow UI won't be reflected in the Azure ML UI.
```
Comment (akarloff) [inline how-to-use-azureml/azure-databricks/README.md]: At the moment, data is only generated in the Azure Machine Learning workspace for tracking.

File: how-to-use-azureml/azure-databricks/README.md [hunk 6]
```diff
@@ -1,33 +1,73 @@
...
+        dbutils.library.installPyPI("azureml-mlflow")
+        dbutils.library.restartPython()  # Removes Python state
+        ```
+    2. [Cluster Libraries](https://docs.azuredatabricks.net/user-guide/libraries.html#install-a-library-on-a-cluster)
+    ![Cluster Library](./img/cluster-library.png)
+2. [Set the MLflow tracking URI](https://mlflow.org/docs/latest/tracking.html#where-runs-are-recorded) to the following scheme:
+    ```
+    adbazureml://${azuremlRegion}.experiments.azureml.net/history/v1.0/subscriptions/${azuremlSubscriptionId}/resourceGroups/${azuremlResourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/${azuremlWorkspaceName}
+    ```
+    1. You can automatically configure this on your clusters using this helper script
```
Comment (akarloff) [inline how-to-use-azureml/azure-databricks/README.md]: script:

File: how-to-use-azureml/azure-databricks/README.md [hunk 7]
```diff
@@ -1,33 +1,73 @@
...
+You can use Azure Databricks as a compute target from [Azure Machine Learning Pipelines](https://docs.microsoft.com/en-us/azure/machine-learning/service/concept-ml-pipelines). Take a look at this notebook for details: [aml-pipelines-use-databricks-as-compute-target.ipynb](https://github.com/Azure/MachineLearningNotebooks/tree/master/how-to-use-azureml/azure-databricks/databricks-as-remote-compute-target/aml-pipelines-use-databricks-as-compute-target.ipynb).
+
+# Linked Azure Databricks and Azure ML Workspaces (Preview)
+Customers can now link Azure Databricks and AzureML Workspaces to better enable MLOps scenarios by [managing their tracking data in a single place when using the MLflow client](https://mlflow.org/docs/latest/tracking.html#mlflow-tracking) - the Azure ML workspace.
+
+## Linking the Workspaces (Admin operation)
+
+1. The Azure Databricks Azure portal blade now includes a new button to link an Azure ML workspace
```
Comment (akarloff) [inline how-to-use-azureml/azure-databricks/README.md]: workspace.

File: how-to-use-azureml/azure-databricks/README.md (hunk 6 above)
Comment (balapv) [inline how-to-use-azureml/azure-databricks/README.md]: Is the init script valid only for this notebook or does it apply to any notebook created in this workspace? If i use the init script, do i still need to use a tracking URI?

Comment (namikhai) [reply how-to-use-azureml/azure-databricks/README.md]: This is a cluster scoped init script so it applies to all notebooks created from that cluster you uploaded the init script to.
If you use that init script you do not need to manually set the tracking uri in the notebook.

Comment (akshaya-a) [reply how-to-use-azureml/azure-databricks/README.md]: @balapv Added some more wording to address your questions - does that help?

Comment (balapv) [reply how-to-use-azureml/azure-databricks/README.md]: Looks good. Thanks

**Pull request merged**